descriptor.go

```diff
@@ -1437,3 +1445,733 @@ func parseDescriptors(i *astikit.BytesIterator) (o []*Descriptor, err error) {
 	}
 	return
 }
+
+// TODO (barbashov) move it to BitsWriter?
+func writeBytesN(b *astikit.BitsWriterBatch, bs []byte, n int, padByte uint8) {
```

Comment:
I think that would be a good idea to move it to BitsWriter

Reply:
Added to BitsWriter: https://github.com/asticode/go-astikit/pull/8

Will change the code in astits once it's merged.

Reply:
I've merged the PR

Reply:
Changed the code